cloudevents/languages/he/README.md

```diff
@@ -0,0 +1 @@
+מסמך זה תרם תורגם. בבקשה תשתמשו [בגרסה האנגלית של המסמך](../../README.md) לבינתיים.
```

Comment:
Google translates this as:
This contributed document has been translated. Please use [the English version of the document](https://github-com.translate.goog/cloudevents/spec/blob/4f1e496fef4383a70fb2d859fd47a8b7ec99afb4/cloudevents/README.md?_x_tr_sl=auto&_x_tr_tl=en&_x_tr_hl=en&_x_tr_pto=wapp) for now.

Shouldn't it be "**not** translated" ? I suspect the others have the same missing "not"

Reply:
Oops, one char typo. Nice catch